pkgs/development/libraries/flutter-engine/deps.nix

```diff
@@ -0,0 +1,736 @@
+{ lib, fetchgit, fetchcipd, fetchFromGitHub, hostPlatform }@args:
+with lib;
+let
+  fetchFromGitHub = { owner, repo, fetchSubmodules ? false, rev ? "HEAD", sha256 ? fakeHash }: args.fetchFromGitHub {
+    inherit owner repo fetchSubmodules rev sha256;
+  };
+
+  fetchFromGoogle = { owner, repo, fetchSubmodules ? false, rev ? "HEAD", sha256 ? fakeHash }: fetchgit {
+    url = "https://${owner}.googlesource.com/${repo}.git";
+    inherit rev sha256 fetchSubmodules;
+  };
+
+  toolchainArch = if hostPlatform.isx86_64 then "amd64" else if hostPlatform.isAarch64 then "aarch64" else throws "Unsupported platform";
+  ## Note: this must match Flutter Engine's DEPS file
+in
+{
+  "src/buildtools/linux-x64/clang" = fetchcipd {
+    package = "fuchsia/third_party/clang/linux-amd64";
+    version = "ugk-KfeqO9fhSfhBFRG4Z-56Kr2AQVSEbku9AEUdotYC";
+    sha256 = "sha256-+ce65wnMJap6sgnpLJwsKneeiuNKVXVLiJWobUxg2Hg=";
+  };
+  "src/flutter/prebuilts/linux-x64/dart-sdk" = fetchcipd {
+    package = "flutter/dart-sdk/linux-amd64";
+    version = "git_revision:b6592742d9f1a82af319f46eda3d59a61e89b91b";
+    sha256 = "sha256-4g3KJYlnf9sD1ej94uyg4R7Ob0eKuzhEnIJcI7qe5cg=";
+  };
+  "src/flutter/third_party/gn" = fetchcipd {
+    package = "gn/gn/\${platform}";
+    version = "git_revision:b79031308cc878488202beb99883ec1f2efd9a6d";
+    sha256 = "sha256-QotGyiQ9Ixgw0Apg7oFX7Cfs4jJKEhbDBo3r+o/4AcY=";
+  };
```

Comment:
source? how did you generate this list of dependencies?
this will be interesting for future updates

Reply:
I've mentioned in the comment that it would be the [DEPS](https://github.com/flutter/engine/blob/857bd6b74c5eb56151bfafe91e7fa6a82b6fee25/DEPS) file in the engine's repo.

Reply:
aah, well hidden

> `## Note: this must match Flutter Engine's DEPS file`

please add a comment in the first line, something like

> `# based on https://github.com/flutter/engine/blob/main/DEPS`

Reply:
Done